src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -87,28 +87,44 @@ export class FeederPodcastModel extends BaseModel {
       this.category = '';
       this.subCategory = '';
     }
+
+    this.explicit = this.doc['explicit'] || '';
+    if (this.explicit) {
+      this.explicit = this.explicit.charAt(0).toUpperCase() + this.explicit.slice(1);
+    }
+
+    this.language = this.doc['language'] || '';
+    if (this.language) {
+      this.language = this.language.toLowerCase();
+    }
+
+    this.publishedUrl = this.doc['publishedUrl'] || '';
+    if (this.doc['url']) {
+      this.publicFeedUrl = this.doc['url'];
+      this.hasPublicFeed = true;
+    }
   }
 
   encode(): {} {
     let data = <any> {};
 
-    // unset things with nulls instead of blank strings
-    data.explicit = this.explicit || null;
-    if (data.explicit) {
-      data.explicit = data.explicit.toLowerCase();
-    }
+    // unset with nulls instead of blank strings
+    data.complete = this.complete;
+    data.copyright = this.copyright || null;
+    data.enclosurePrefix = this.enclosurePrefix || null;
+    data.language = this.language || null;
     data.link = this.link || null;
     data.newFeedUrl = this.newFeedUrl || null;
     data.url = this.publicFeedUrl || null;
 
-    if (this.authorName || this.authorEmail) {
-      data.author = {
-        name: this.authorName,
-        email: this.authorEmail
-       };
-    }
-
-    data.enclosurePrefix = this.enclosurePrefix || null;
+    ['author', 'owner', 'managingEditor'].forEach((role) => {
+      if (this[`${role}Name`] || this[`${role}Email`]) {
```

Comment:
If these fields are empty string, the old values are retained. You can clear one of the fields, name or email, but not both. There's an existing bug ticket on this for authorEmail #343. We can either fix here or update that ticket with these additional fields.